src/Refactoring-UI/RBRenameVariableDriver.class.st

```diff
@@ -0,0 +1,88 @@
+"
+I am an interaction class to rename both instance and class variables
+"
+Class {
+	#name : 'RBRenameVariableDriver',
+	#superclass : 'RBInteractionDriver',
+	#instVars : [
+		'variable',
+		'newName',
+		'oldName',
+		'shouldEscape'
+	],
+	#category : 'Refactoring-UI-Drivers',
+	#package : 'Refactoring-UI',
+	#tag : 'Drivers'
+}
+
+{ #category : 'class name' }
+RBRenameVariableDriver class >> rename: aClyVariable [
+
+	^ self new
+		  variable: aClyVariable;
+		  yourself
+]
+
+{ #category : 'execution' }
+RBRenameVariableDriver >> classRefactoring [
+	^ RBRenameInstanceVariableRefactoring
+]
+
+{ #category : 'resources' }
+RBRenameVariableDriver >> configureRefactoring [
+
+	refactoring := variable createVariableRefactoring:
+		               self classRefactoring
+]
+
+{ #category : 'accessing' }
+RBRenameVariableDriver >> model: aRBBrowserEnvironment [ 
+	model := aRBBrowserEnvironment
```

Comment:
There is no model in the superclass? Looks strange. We will have to pull it up

Reply:
The instance variable model exists in `RBInteractionDriver`, but the setter didn't. I just pulled the method model: up